packages/react-components/react-virtualizer/src/components/VirtualizerScrollView/useVirtualizerScrollView.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { resolveShorthand } from '@fluentui/react-utilities';
+import { useVirtualizer_unstable } from '../Virtualizer/useVirtualizer';
+import { VirtualizerScrollViewProps, VirtualizerScrollViewState } from './VirtualizerScrollView.types';
+
+export function useVirtualizerScrollView_unstable(
+  props: VirtualizerScrollViewProps,
+  virtualizerLength: number,
+): VirtualizerScrollViewState {
+  const virtualizerState = useVirtualizer_unstable({ ...props, virtualizerLength });
+
+  const setScrollRef = React.useCallback(
+    (element: HTMLDivElement) => {
+      if (!element || !props.scrollViewRef || props.scrollViewRef.current === element) {
+        return;
+      }
+      props.scrollViewRef.current = element;
+    },
+    [props.scrollViewRef],
+  );
+
+  return {
+    ...virtualizerState,
+    components: {
+      ...virtualizerState.components,
+      container: 'div',
+    },
+    container: resolveShorthand(props.container, {
+      required: true,
+      defaultProps: {
+        ref: setScrollRef,
+        role: 'list',
```

Comment:
I don't think it makes sense to set a `list` role for a container of any children that could be virtualized

Reply:
I figured having this as a default would be the most common use case for a virtualized scrollview that users can override, but agreed we don't want to assume, I'll remove and add via the demos to be sure its A11Y compliant still.